ElementX/Sources/Screens/SecureBackup/SecureBackupLogoutConfirmationScreen/View/SecureBackupLogoutConfirmationScreen.swift

```diff
@@ -0,0 +1,135 @@
+//
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Combine
+import Compound
+import SwiftUI
+
+struct SecureBackupLogoutConfirmationScreen: View {
+    @ObservedObject var context: SecureBackupLogoutConfirmationScreenViewModel.Context
+    
+    @ScaledMetric private var iconSize = 70
+    
+    var body: some View {
+        NavigationStack {
+            VStack(spacing: 16) {
+                header
+                content
+                Spacer()
+                footer
+            }
+            .padding()
+            .toolbar { toolbar }
+            .background(Color.compound.bgCanvasDefault.ignoresSafeArea())
+        }
+    }
+    
+    @ViewBuilder
+    private var header: some View {
+        HeroImage(image: Image(asset: Asset.Images.secureBackupOff))
+    }
+    
+    @ViewBuilder
+    private var content: some View {
+        Text(title)
+            .foregroundColor(.compound.textPrimary)
+            .font(.compound.headingMDBold)
+            .multilineTextAlignment(.center)
+        
+        Text(subtitle)
+            .foregroundColor(.compound.textSecondary)
+            .font(.compound.bodyMD)
+            .multilineTextAlignment(.center)
+        
+        if context.viewState.mode == .backupOngoing {
+            Spacer()
+            ProgressView()
+        }
+    }
+    
+    @ViewBuilder
+    private var footer: some View {
+        if context.viewState.mode == .saveRecoveryKey {
+            Button {
+                context.send(viewAction: .settings)
+            } label: {
+                Text(L10n.commonSettings)
+            }
+            .buttonStyle(.compound(.primary))
+        }
+        
+        Button(role: .destructive) {
+            context.send(viewAction: .logout)
+        } label: {
+            Text(L10n.actionSignout)
```

Comment:
Perhaps for your quest to <del>signout</del>
```suggestion
            Text(L10n.actionLogout)
```

Reply:
I cannot unfortunately, it's defined as `Sign out` in the designs. I'm happy as long as **we** call it logout everywhere

Reply:
😂 woah I totally didn't see this was a string!